tests/unit/domain/test_imports.py

```diff
@@ -76,3 +76,13 @@ def test_object_representation(self):
     )
     def test_string_object_representation(self, test_object, expected_string):
         assert str(test_object) == expected_string
+
+
+class TestImportExpression:
```

Comment:
It would be good to have a test for `has_wildcard_expression` here.

Reply:
done